### DISCUSSION TITLE
Blu-ray Supporting 1080p 24 Files

### DISCUSSION THREAD

**Discussion by donnafaye0323** [justdan96/tsMuxer#754] [General]
Most movies are shot at a frame rate of 24 frames per second. It is for this reason that the 1080p/24 resolution has become the de facto standard of Blu-ray movies, but there is a lot more to it than that.

This is why I import 1080p files on Tsmuxer, and it would work as well. 

Here's a wonderful article about why blu-ray discs support 1080p 24fps:

https://hometheaterreview.com/1080p24/

This will really help is alot.

**Comment by ValeZAA** [comment]
Actually only some of Blu-rays are 24p. Most of them are 24/1.001p.
